danceradvent/public/articles/2023/15-mason-whats-old-is-new-again.pod

```diff
@@ -0,0 +1,133 @@
+=head1 Dancer2::Plugin::Mason - What's old is new again!
+
+This year, Retro Santa paid us a visit and gave us a new gift of something
+old - an L<HTML::Mason> plugin for Dancer2!
+
+=head2 But Santa, why? WHY?
+
+Mason is a solid templating engine, but it lacks a controller. A
+number of projects have built their own controllers (such as
+L<Request Tracker (RT)|https://bestpractical.com/request-tracker>), and
+others struggle without one. That's where Dancer2 comes in.
+
+L<Dancer2::Template::Mason> is a template engine for HTML::Mason. If
+HTML::Mason is your favorite template engine, you can now use it
+in your favorite web framework. If you have legacy Mason applications
+that would benefit from having a proper controller, or a general
+refresh with some a more modern design, Dancer2 has your back.
+
+=head2 Does this mean I'm NOT on the Naughty List?
+
+Not at all! It's actually a pretty good gift, combining two powerful tools
+that are already in the toolbox of a lot of Perl developers. Go ahead,
+you can thank Retro Santa now. I'll wait.
+
+HTML::Mason, like all frameworks, has its warts and annoyances, but also
```

Comment:
Like most of us, you have the natural tendency to apologize for the flaws of the thing before you (humbly) sing its praises. We're selling wares here. Sing loudly first, and maybe mention the flaws when it's necessary. Here, I think we can remove the paragraph altogether.

Reply:
Great point!